src/components/Main.ui.tsx

```diff
@@ -388,9 +384,8 @@ export class MainComponent extends React.Component<Props> {
                 key="pluginViewBuy"
                 component={withNavigation(ifLoggedIn(GuiPluginViewScene))}
                 navTransparent
-                renderTitle={props => <HeaderTitle title={props.route.params.plugin.displayName} />}
-                // @ts-expect-error
-                renderLeftButton={renderPluginBackButton()}
+                renderTitle={props => <HeaderTitle navigation={props.navigation} title={props.route.params.plugin.displayName} />}
+                renderLeftButton={renderPluginBackButton}
```

Comment:
If `renderPluginBackButton` is a component, it should be moved to the `components/` dir and be renamed using pascal-casing.